pt/lessons/basics/mix.md

```diff
@@ -130,8 +130,20 @@ Mix, bem como Bundler, suporta ambientes diferentes. Naturalmente mix trabalha c
 + `:test` — Usado por `mix test`. Coberto futuramente na nossa próxima lição.
 + `:prod` — Usado quando nós enviamos a nossa aplicação para produção.
 
-O ambiente atual pode ser acessado usando `Mix.env`. Como esperado, o ambiente pode ser alterado através da variável de ambiente `MIX_ENV`:
+Como esperado, o ambiente pode ser alterado através da variável de ambiente `MIX_ENV`:
```

Comment:
Hi @VacariGabriel , the translations try to follow the original (english) text. 

Please propose any changes and updates to the text to the [english version](https://github.com/elixirschool/elixirschool/blob/master/en/lessons/basics/mix.md) as it is the main text of elixirschool.